File: GARI_Umbria_2026-08-03/GiasNG/src/app/Model/anagrafiche/ParticelleCatastaliZona.ts
import { BaseCodeDescr } from '../baseClass/baseCodeDescr';
import { TitoloDiPossesso } from '../metaschema/TitoloDiPossesso';
import { IntervalloTemporale } from './IntervalloTemporale';

export class ParticelleCatastaliZona {
    zona: BaseCodeDescr;
    Area: number;
    validita: IntervalloTemporale;

}

File: GARI_Umbria_2026-08-03/DotNet/AgronicaCoreStd/AgronicaCoreModelsSTD/anagrafiche/ParticelleCatastaliZona.cs
using AgronicaCoreModelsSTD.baseClass;
using System;
using System.Collections.Generic;
using System.Text;

namespace AgronicaCoreModelsSTD.anagrafiche
{
    public class ParticelleCatastaliZona
    {

        public BaseCodeDescr zona { get; set; }
        public double Area { get; set; }
        public IntervalloTemporale validita { get; set; }

        public ParticelleCatastaliZona()
        {

        }
    }
}
